pkgs/development/interpreters/bqn/mlochbaum-bqn/default.nix

```diff
@@ -7,13 +7,13 @@
 
 stdenvNoCC.mkDerivation rec {
   pname = "bqn";
-  version = "0.pre+date=2023-05-09";
+  version = "0.pre+date=2023-05-17";
```

Comment:
Please change this to the `unstable-YYYY-MM-DD` format.

Reply:
Done.